docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Headless Login (Preview)
+description: Headless Login
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Headless login is currently in Preview mode.
+</Admonition>
+
+Headless login provides Teleport users with a secure way to authenticate with WebAuthn from a remote machine. This enables use cases which usually are not compatible with WebAuthn features, such as:
+
+- Logging into Teleport with WebAuthn from a remote dev box
+- Connecting to a Teleport Node from a remote dev box with Per-session MFA 
+- Performing `tsh scp` from one Teleport Node to another with Per-session MFA
+- Logging into Teleport on a machine without a WebAuthn compatible browser
+
+<Admonition type="note" title="Headless Login Support">
+  Headless login only supports following `tsh` commands:
+  
+  - `tsh ls`
+  - `tsh ssh`
+  - `tsh scp`
+
+  In the future, headless login may be extended to other `tsh` commands.
+</Admonition>
+
+## Prerequisites
```

Comment:
Can you also add a note that min supported version is v12.2, so headless doesn't work with every v12 release?